test/distributed/test_nccl.py

```diff
@@ -17,6 +18,7 @@
     TestCase = object  # noqa: F811
 
 
+@skipCUDAIfRocm
```

Comment:
I don't think this will work. You should import TEST_WITH_ROCM from torch.testing._internal.common_utils and annotate each failing test:

```
@unittest.skipIf(TEST_WITH_ROCM, 'Test is failing on ROCm, see issue <link to issue here>')
```